.github/workflows/publish.yml

```
@@ -0,0 +1,82 @@
+        uses: actions/upload-artifact@v4
+        with:
+          name: jarvix-images
+          path: |
+            jarvix-engine-${{ github.sha }}.tar.gz
+            jarvix-app-${{ github.sha }}.tar.gz
+
+  helm-deploy:
+    name: Helm deploy (optional)
+    runs-on: ubuntu-latest
+    needs: build-and-push
+    if: ${{ secrets.KUBE_CONFIG != '' }}
```

actionlint (1.7.10)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)